source/javascripts/gradient_fade.js

```diff
@@ -0,0 +1,16 @@
+function init() {
+    // Create the listener function
+    var updateLayout = _.debounce(function(e) {
```

Comment:
This was an attempt at a solution to the problem that 1. we want to have the sticky header which means that the header should have a solid white background to prevent text from bleeding through, so the gradient needs to disappear when the header starts to go 'over' the text, otherwise we get this is if we just set the gradient once:

![screen shot 2015-06-04 at 3 18 22 pm](https://cloud.githubusercontent.com/assets/7738663/7992391/f8f94da2-0acc-11e5-94aa-c40aaef0da49.png)